crates/assists/src/handlers/extract_struct_from_enum_variant.rs

```diff
@@ -37,9 +37,24 @@ pub(crate) fn extract_struct_from_enum_variant(
         ast::StructKind::Tuple(field_list) => field_list,
         _ => return None,
     };
+    // Grab the tuple field inner type as a string (if it exists and is alone)
+    // to check against structs that may have a differing name from the enum
+    // variant itself
+    let variant_inner = match variant.field_list() {
+        Some(FieldList::TupleFieldList(tfl)) => {
```

Comment:
You can skip the `Option` matching here by using the `field_list` from above, line 36.

Reply:
oh wow, not sure how I totally missed that, thanks!